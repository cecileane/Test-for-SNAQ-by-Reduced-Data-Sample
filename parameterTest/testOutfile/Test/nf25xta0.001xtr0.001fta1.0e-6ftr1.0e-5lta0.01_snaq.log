optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 0.01.
rootname for files: nf25xta0.001xtr0.001fta1.0e-6ftr1.0e-5lta0.01_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 11512
seed: 11512 for run 1
Sat Oct  1 17:32:02 2016

 BEGIN SNaQ for run 1, seed 11512 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 56 (failures: 25) with -loglik=126.74251 and ht_min=[0.2856,10.0,0.48942,0.51916,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 12		 12		 3	 |	 0		 0		 --
mvtarget	 13		 13		 1	 |	 0		 0		 --
chdir		 14		 14		 3	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 16		 11		 0	 |	 6		 6		 --
Total		 56		 51		 8	 |	 6		 6		 4
Proportion	 -- 		 1.0		 0.2	 |	 -- 		 1.0		 0.7

 FINISHED SNaQ for run 1, -loglik of best 126.74250981975364
((((5,(#H7:::0.28560309151319346,6):10.0):0.5191606292429678,3):0.48941892526983916,4):1.5121336861545507,2,(1)#H7:::0.7143969084868065);
---------------------
seed: 26739 for run 2
Sat Oct  1 17:32:06 2016

 BEGIN SNaQ for run 2, seed 26739 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 47 (failures: 25) with -loglik=126.74251 and ht_min=[0.2856,0.48942,1.51213,10.0,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 7		 7		 0	 |	 0		 0		 --
mvtarget	 13		 13		 0	 |	 0		 0		 --
chdir		 7		 7		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 18		 16		 1	 |	 5		 5		 --
Total		 47		 44		 4	 |	 5		 5		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 126.74250981974929
(3,((2,(1)#H7:::0.7143969538016022):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7:::0.2856030461983978):10.0):0.5191607417655206);
---------------------

Sat Oct  1 17:32:10 2016
MaxNet is (3,((2,(1)#H7:::0.7143969538016022):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7:::0.2856030461983978):10.0):0.5191607417655206); 
with -loglik 126.74250981974929
