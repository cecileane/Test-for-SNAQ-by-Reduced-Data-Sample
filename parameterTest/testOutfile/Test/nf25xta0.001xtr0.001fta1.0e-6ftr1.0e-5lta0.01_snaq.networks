(3,((2,(1)#H7:::0.7143969538016022):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7:::0.2856030461983978):10.0):0.5191607417655206);, with -loglik 126.74250981974929 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((2,(1,#H7:::0.3070770183819041):7.546271557318668):0.661205369482845,4):0.4642645749751077,(5,(6)#H7:::0.692922981618096):1.6805328335848715);, with -loglik 128.24190466681512
(3,(4)#H7:::0.608941756284198,(5,(6,(1,(#H7:::0.391058243715802,2):3.478860184498319):1.1805724802834945):0.22007714122045652):0.021048613992158148);, with -loglik 172.1289155322852
(3,(#H7:::0.4182421177412259,4):1.8866760993626825,(5,(6,(1,(2)#H7:::0.5817578822587741):3.837087769997838):0.16886078475485478):0.026892580268853418);, with -loglik 192.29252807536943
(((3)#H7:::0.5964348356796727,4):0.06924069158959822,2,(1,(6,(5,#H7:::0.40356516432032735):9.730016871337936):0.9927754217172703):0.09755017995665403);, with -loglik 215.12838918911956
(3,((2,(1,(6,#H7:::0.49240197840356165):4.569266609291792):0.04135979916196679):0.20281462131953834,4):1.1978887199968355,(5)#H7:::0.5075980215964384);, with -loglik 231.46451882429864
