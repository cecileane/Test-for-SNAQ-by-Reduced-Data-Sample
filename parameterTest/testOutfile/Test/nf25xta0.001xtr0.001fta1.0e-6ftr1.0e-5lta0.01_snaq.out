(3,((2,(1)#H7:::0.7143969538016022):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7:::0.2856030461983978):10.0):0.5191607417655206); -Ploglik = 126.74250981974929
 Dendroscope: (3,((2,(1)#H7):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7):10.0):0.5191607417655206);
 Elapsed time: 7.874880107 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,((2,(1)#H7:::0.7143969538016022):1.5121336394054226,4):0.4894188330261261,(5,(6,#H7:::0.2856030461983978):10.0):0.5191607417655206);, with -loglik 126.74250981974929
 ((((5,(#H7:::0.28560309151319346,6):10.0):0.5191606292429678,3):0.48941892526983916,4):1.5121336861545507,2,(1)#H7:::0.7143969084868065);, with -loglik 126.74250981975364
-------
